frame/dapps-staking/src/lib.rs

```diff
@@ -629,6 +617,37 @@ decl_module! {
                 }
             }
         }
+
+        #[weight = SimpleDispatchInfo::FixedNormal(1_000)]
+        fn claim(origin, era: EraIndex) {
```

Comment:
Could you add documents comment about `claim` methods?

Reply:
modified it in c2de840